modules/storages/spec/requests/api/v3/storages/storages_spec.rb

```diff
@@ -343,4 +446,61 @@ def query(_)
       end
     end
   end
+
+  describe 'POST /api/v3/storages/:storage_id/oauth_client_credentials' do
```

Comment:
The GET endpoint is missing request specs. 

If this file gets unwieldy, it could be split up into single files as is e.g. done [for notifications](https://github.com/opf/openproject/tree/dev/spec/requests/api/v3/notifications)

Reply:
there is no GET on that route